source/_integrations/samsungtv.markdown

```diff
@@ -126,6 +126,7 @@ For example: for model `UN55NU7100`, the `UN55` would mean it's an LED, North Am
 - U6300
 - RU7100
 - RU7172
+- qe55q90ratxzt
```

Comment:
Could you please remove the 4 characters from it? As per documentation:

> The first letter (U, P, L, H & K) represent the screen type, e.g., LED or Plasma. The second > letter represents the region, E is Europe, N is North America and A is Asia & Australia. The > two numbers following that represent the screen size. If you add your model remember to remove these first 4 characters before adding to the list.

Furthermore, please capitalize the letters.

Thanks! 👍